### ISSUE TITLE
Create new version of UGAMarineQry.js to pull GeoJSON from S3

### ISSUE THREAD

**Issue by metanomy** [metanomy/tampabaymap#6]
@simus81 I need a quick spot of work done by Tuesday (and am willing to pay accordingly).

I need a new version of your UGAMarineQry.js script that can pull individual GeoJson files (see attached file as an example) from our S3 bucket (https://console.aws.amazon.com/s3/home?region=us-west-2#&bucket=marinedebris&prefix=) and write them into our Geoserver (dev.tampabaymap.org:8181/geoserver) to the geonode:metanomy_marine_debris layer using WFS-T. 

[2016_03_11_13_29_18-plastic.json.zip](https://github.com/metanomy/tampabaymap/files/170636/2016_03_11_13_29_18-plastic.json.zip)

**Comment by simus81** [comment]
Sorry the dead line is to short,
maybe I can do it but I will need  the proprer autorisation to the amz s3
bucket and the Geoserver.

2016-03-12 22:40 GMT+01:00 metanomy notifications@github.com:

**Comment by metanomy** [comment]
I figured out an easier way to do it - use ogr2ogr to import directly into postgresql. 

I'm going to open a new issue, though, so I can get your UGAMarineQry.js working with my new environment.

**Comment by simus81** [comment]
So, still need my help?
Anyway , i can work on my script for you (with plaisure) but please let me
know a little bit earlier. (at least one week).

2016-03-14 12:48 GMT+01:00 metanomy notifications@github.com:

**Comment by metanomy** [comment]
Yes, sir, I do need your help - just to get the existing script working. I normally wouldn't ask with such short notice, but we have a last-minute presentation this Wednesday, and we'd like to show off your work :)

**Comment by simus81** [comment]
ok. i have a lot of work from my side but i will try to do my best to help
you.

2016-03-14 13:36 GMT+01:00 metanomy notifications@github.com:
